src/main/java/io/kestra/plugin/fs/ssh/Command.java

```diff
@@ -0,0 +1,103 @@
+package io.kestra.plugin.fs.ssh;
+
+import com.jcraft.jsch.ChannelExec;
+import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.Session;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.Task;
+import io.kestra.core.models.tasks.VoidOutput;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.fs.vfs.AbstractVfsInterface;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.io.ByteArrayOutputStream;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Send a command to a remote server using SSH."
+)
+@Plugin(
+    examples = {
+        @Example(
+            code = {
+                "host: localhost",
+                "port: 22",
+                "username: foo",
+                "password: pass",
+                "command: ls",
+            }
+        )
+    }
+)
+public class Command extends Task implements AbstractVfsInterface, RunnableTask<VoidOutput> {
+    private static final long SLEEP_DELAY = 25L;
+
+    private String host;
+    private String username;
+    private String password;
+    @Builder.Default
+    private String port = "22";
+
+    @Schema(title = "The command to run on the remote server")
+    @PluginProperty(dynamic = true)
+    private String command;
+
+    @Schema(title = "Whether to check if the host public key could be found among known host, one of 'yes', 'no', 'ask'")
+    @PluginProperty
+    @Builder.Default
+    private String strictHostKeyChecking = "no";
+
+    @Override
+    public VoidOutput run(RunContext runContext) throws Exception {
+        Session session = null;
+        ChannelExec channel = null;
+
+        try(var outStream = new ByteArrayOutputStream(); var errStream = new ByteArrayOutputStream()) {
+            var renderedHost = runContext.render(host);
+            var renderedPort = runContext.render(port);
+
+            session = new JSch().getSession(runContext.render(username), renderedHost, Integer.parseInt(renderedPort));
+            session.setPassword(runContext.render(password));
+            session.setConfig("StrictHostKeyChecking", strictHostKeyChecking);
+            session.connect();
+
+            channel = (ChannelExec) session.openChannel("exec");
+            channel.setCommand(runContext.render(command));
```

Comment:
Can you use multiline commands like on bash?

Reply:
No, you set a command, you establish a connection, you wait untill the server disconnect.
If you want to execute a script, you had to upload it first (via SFTP for ex), then use the SSH task to execute it.

It's not like a batch, it's a way to execute a command on a remote server.

Reply:
I want to have a common interface on all bash task, the current one on bash only concat [the command](https://github.com/kestra-io/kestra/blob/0a542a884f1f91ff70cb1d8dd75ba182371f6b55/core/src/main/java/io/kestra/core/tasks/scripts/Bash.java#L131), it's better than a simple string